e2e/suite_test.go

```diff
@@ -15,10 +19,18 @@ import (
 	"github.com/barkimedes/go-deepcopy"
 )
 
+type getNodeBootstrapping func(ctx context.Context, nbc *datamodel.NodeBootstrappingConfiguration) (*datamodel.NodeBootstrapping, error)
```

Comment:
maybe move this to another file entirely? could make a new one called nodebootstrapping.go or nbc.go

Reply:
might be able to get away without this with the other changes I suggested, either way if you wanna keep it I'd rename to getNodeBootstrappingFn or getNodeBootstrappingFunction or something